recipes/default.rb

```diff
@@ -108,4 +112,16 @@
   mode '0755'
   owner 'root'
   only_if { node['anaconda']['system_path'] }
+  action :nothing
+end
+
+file marker_file do
+    content "Do not modify or remove. This is a marker file for CHEF"
```

Comment:
Indentation is off, should be 2 spaces.

Reply:
Oops... My editor is configured for 4 spaces with no tabs.
I'll correct.